iron-overlay-behavior.html

```diff
@@ -167,10 +167,33 @@
       return this._focusedChild || Polymer.dom(this).querySelector('[autofocus]') || this;
     },
 
+    /**
+     * Nodes that can receive focus (overlay included). They will be ordered by their tabindex.
+     * @type [Node]
+     */
+    get _focusableNodes() {
+      var focusables = Polymer.dom(this).querySelectorAll('a[href]:not([tabindex="-1"]),area[href]:not([tabindex="-1"]),input:not([disabled]):not([tabindex="-1"]),select:not([disabled]):not([tabindex="-1"]),textarea:not([disabled]):not([tabindex="-1"]),button:not([disabled]):not([tabindex="-1"]),iframe:not([tabindex="-1"]),[tabindex]:not([tabindex="-1"]),[contentEditable=true]:not([tabindex="-1"])');
```

Comment:
`paper-button` has `tabindex=0` which is selected via `[tabindex]:not([tabindex="-1"])` (towards the end of this line)